install.sh

```diff
@@ -1,5 +1,10 @@
 #! /usr/bin/env bash
 
+if [[ "${BASH_VERSION%%.*}" -eq 3 ]]; then
+    echo "Atuin doesn't work on bash 3.2. Please upgrade your version of bash (possibly via homebrew or ports) and try again"
+	exit
```

Comment:
The previous suggestion was just discarded after the application. I'm not sure if that is intentional, but I think at least the current PR needs updating. @fragmede Could you take a look at it again?

```suggestion
    echo "Atuin has limited support for Bash 3.2. The config enter_accept cannot be turned off." >&2
    echo "To turn off enter_accept, please upgrade your version of bash (possibly via homebrew or ports)" >&2
```

Reply:
Hows 3a3ce3eee333 look?

Reply:
LGTM